web/packages/core/src/shadow-template.ts

```diff
@@ -218,13 +222,91 @@ ruffleShadowTemplate.innerHTML = `
+        <svg class="loadingAnimation" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 66 66">
+            <g class="loading">
```

Comment:
The `loading` group doesn't seem to have any effect. Please remove it, and its CSS styling as well.